app/router.go

```diff
@@ -104,7 +104,7 @@ var topologyRegistry = map[string]topologyView{
 	"applications": {
 		human:    "Applications",
 		parent:   "",
-		renderer: render.FilterUnconnected{Renderer: render.ProcessRenderer},
+		renderer: render.FilterUnconnected{Renderer: render.ProcessWithContainerNameRenderer{}},
```

Comment:
Is there a reason we wouldn't just have this be the default behavior of ProcessRenderer?